test_util/integration_test.py

```diff
@@ -1260,10 +1260,12 @@ def wait_for_endpoint():
 
     wait_for_endpoint()
 
-    test_cache_url = "http://{}:{}/signal_test_cache".format(service_points[0].host, service_points[0].port)
+    test_host = service_points[0].host
+    test_port = service_points[0].port
+    test_cache_url = "http://{}:{}/signal_test_cache".format(test_host, test_port)
+
     cmd = """
-data=`/opt/mesosphere/bin/dcos-signal -report-host leader.mesos -test 2> /dev/null`;
-curl -H 'Content-Type: application/json' -X POST -d "$data" {};
+/opt/mesosphere/bin/dcos-signal -master-url http://leader.mesos -test-url {}  2> /dev/null;
```

Comment:
It runs the POST to stdout if you execute it with -v, I've also left the print statement in the test code so we can see the return. However, since this is executed via marathon, getting the stdout from the service being executed is rather difficult.